srcpy/py_finger_types.cpp

```diff
@@ -61,7 +61,8 @@ PYBIND11_MODULE(py_finger_types, m)
         .def("get_time_stamp_ms", &FingerTypes::Frontend::get_time_stamp_ms)
         .def("append_desired_action", &FingerTypes::Frontend::append_desired_action)
         .def("wait_until_time_index", &FingerTypes::Frontend::wait_until_timeindex)
-        .def("get_current_time_index", &FingerTypes::Frontend::get_current_timeindex);
+        .def("get_current_time_index", &FingerTypes::Frontend::get_current_timeindex)
+        .def("get_oldest_time_index", &FingerTypes::Frontend::get_oldest_timeindex);
```

Comment:
Please also add this in `py_one_joint_types.hpp` and `py_two_joint_types.hpp`
This duplication is a bit annoying, I already have plans to improve this but for now we should keep the three files in sync.